core/templates/dev/head/pages/base.html

```diff
@@ -165,6 +165,11 @@ <h2>
 
     {% include 'pages/footer_js_libs.html' %}
 
+    <script src="/templates/dev/head/pages/topic-landing-page/topic-landing-page.module.js"></script>
+    <script src="/templates/dev/head/pages/topic-landing-page/topic-landing-page.controller.js"></script>
+    <script src="/templates/dev/head/pages/topic-landing-page/topic-landing-page-stewards/topic-landing-page-stewards.module.js"></script>
+    <script src="/templates/dev/head/pages/topic-landing-page/topic-landing-page-stewards/topic-landing-page-stewards.controller.js"></script>
```

Comment:
Load the dependency scripts before the dependent scripts. A better order would be:
```
    <script src="/templates/dev/head/pages/topic-landing-page/topic-landing-page-stewards/topic-landing-page-stewards.module.js"></script>
    <script src="/templates/dev/head/pages/topic-landing-page/topic-landing-page-stewards/topic-landing-page-stewards.controller.js"></script>
    <script src="/templates/dev/head/pages/topic-landing-page/topic-landing-page.module.js"></script>
    <script src="/templates/dev/head/pages/topic-landing-page/topic-landing-page.controller.js"></script>
```
Here and elsewhere.

Reply:
ok

Reply:
done